client/my-sites/checkout/upsell-nudge/professional-email-upsell/style.scss

```diff
@@ -163,25 +163,21 @@
 	.form-text-input-with-affixes {
 		margin-top: 5px;
 	}
-}
 
-.professional-email-upsell__form-buttons {
-	display: flex;
-	flex-direction: column-reverse;
-	justify-content: space-between;
+	.new-mailbox-list__separator {
+		display: none;
+	}
 
-	@include break-mobile {
-		flex-direction: row;
+	.new-mailbox-list__supplied-actions.not-show-add-new-mailbox {
```

Comment:
I think it'd be nice to use a BEM modifier class here instead. `.new-mailbox-list__supplied-actions_spread` or something ought to work.

It would require a change in `client/my-sites/email/form/mailboxes/components/new-mailbox-list/style.scss` as well, but I believe we could actually remove the `&.not-show-add-new-mailbox` rule there altogether, no?

Reply:
TIL more about the BEM ....